tests/ios/Podfile.lock

```diff
@@ -848,14 +848,14 @@ SPEC CHECKSUMS:
   RNFBFunctions: 325b8be36afad5a424d43b8f39e379eaee0b86be
   RNFBIid: bb061f08b96880eb5fd599c06a92cfe2c164a5e7
   RNFBInAppMessaging: bb5ac95b411f46755ffbc425566822afc0bff78f
-  RNFBMessaging: 6bdab64e89a79afb36f58d75807166e574c489e8
+  RNFBMessaging: e06217e17539a3328e5f06ca34a83010a73af384
   RNFBMLNaturalLanguage: 901aa0ecffbaeca9a26cc38d6dda3a6a2f2dd8ec
   RNFBMLVision: 46a088cab0f986f3359d9340d416e998f0458379
-  RNFBPerf: 20a0ab92075834bb4c88537677b2fffcb023b04c
-  RNFBRemoteConfig: fb46148a3b985697f3538b8c4d3e586a14bb4570
+  RNFBPerf: 3d185f43c89d4159649c6a35813c337857a2e941
+  RNFBRemoteConfig: 96917b7defbb474ffc774f43454686de7df09a9b
   RNFBStorage: ee1bafd9e83d30feaa4af266f2782f109284be9a
   Yoga: 3ebccbdd559724312790e7742142d062476b698e
 
 PODFILE CHECKSUM: 250927a7e70dee93adbb105e4000258dc7be161b
 
-COCOAPODS: 1.9.3
+COCOAPODS: 1.9.1
```

Comment:
You should update your local cocoapods ;-)

Reply:
done :)